documents/ANDROID.md

```diff
@@ -0,0 +1,96 @@
+  * [Consuming from Maven](#consuming-from-maven)
+  * [Consuming from locally installed](#consuming-from-locally-installed)
+* [Samples App](#samples-app)
+
+## Installation
+
+### Minimum requirements
+* Java 11+ ([Download and Install Java](https://www.java.com/en/download/help/download_options.html))
+  * [Set JAVA_HOME](./documents/PREREQUISITES.md#set-java_home)
+* Gradle 7.4.2 ([Download and Install Gradle](https://gradle.org/install/))
+* Android SDK 26 ([Doanload SDK Manager](https://developer.android.com/tools/releases/platform-tools#downloads))
+  * [Set ANDROID_HOME](./documents/PREREQUISITES.md#set-android_home)
+
+### Build and install IoT Device SDK from source
+Supports API 26 or newer.
+NOTE: The shadow sample does not currently complete on android due to its dependence on stdin keyboard input.
+
+``` sh
+# Create a workspace directory to hold all the SDK files
+mkdir sdk-workspace
+cd sdk-workspace
+# Clone the SDK repository
+# (Use the latest version of the SDK here instead of "v1.17.0)
+git clone --branch v1.17.0 --recurse-submodules https://github.com/aws/aws-iot-device-sdk-java-v2.git
+# Compile and install the SDK for Android
+cd aws-iot-device-sdk-java-v2/android
+./gradlew build
+# Install SDK locally
+./gradlew publishToMavenLocal
+```
+
+## Consuming IoT Device SDK Android
+
+### Consuming from Maven
+##### NOTE: aws-iot-device-sdk-android is not currently available in Maven but will be shortly. For now, please publish to maven locally and [Consume from locally installed](#consuming-from-locally-installed)
+Consuming this SDK via Maven is the preferred method of consuming it and using it within your application. To consume
+IoT Device SDK Android in your application, add the following to your `build.gradle` repositories and dependencies:
+
+``` groovy
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    api 'software.amazon.awssdk.iotdevicesdk:aws-iot-device-sdk-android:1.17.0'
+}
+```
+Replace `1.17.0` in `software.amazon.awssdk.iotdevicesdk:aws-iot-device-sdk-android:1.17.0` with the latest release version for the SDK.
+Look up the latest SDK version here: https://github.com/aws/aws-iot-device-sdk-java-v2/releases
+
+### Consuming from locally installed
+You may also consume IoT Device SDK Android in your application using a locally installed version by adding the
+following to your `build.gradle` repositories and depenencies:
+``` groovy
+repositories {
+    mavenLocal()
+}
+
+dependencies {
+    api 'software.amazon.awssdk.iotdevicesdk:aws-iot-device-sdk-android:1.17.0'
+}
+```
+Replace `1.17.0` in `software.amazon.awssdk.iotdevicesdk:aws-iot-device-sdk-android:1.17.0` with the latest release version for the SDK
+or replace with `1.0.0-SNAPSHOT` to use the SDK built and installed from source.
+Look up the latest SDK version here: https://github.com/aws/aws-iot-device-sdk-java-v2/releases
+
+## Samples App
+[Android IoT Samples App README](./samples/Android/README.md)
```

Comment:
I got 404 here

Reply:
I'm getting the same. Best guess is they 404 until they've been added once? I'm pretty sure they're pointing to the correct location.